corehq/apps/reports/static/reports/js/aggregate_user_status.js

```diff
@@ -1,6 +1,5 @@
-/* globals d3, nv */
 hqDefine("reports/js/aggregate_user_status", function () {
-    function aggregateTooltip(key, x, y, e, graph) {
+    function aggregateTooltip(key, x, y, e) {
```

Comment:
So none of the calling code passes in graph?

Reply:
It probably does get passed in - ultimately this function is used as the [tooltipContent option in nvd3](https://nvd3-community.github.io/nvd3/examples/documentation.html) (not a great link since it just says `tooltipContent` is deprecated, but you get the idea) -  but it doesn't matter since the function doesn't use it.